.github/workflows/code-quality.yml

```
@@ -8,18 +8,28 @@ on:
     branches:
       - main
       - dev
+
 jobs:
-  quality:
+  linting:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
       - uses: ./.github/actions/setup
```

actionlint (1.7.4)

name is required in action metadata "/home/jailuser/git/.github/actions/setup/action.yml"

(action)